spec/services/sync_ndbn_members_spec.rb

```diff
@@ -0,0 +1,53 @@
+describe SyncNDBNMembers do
+  describe ".sync" do
+    subject { -> { described_class.sync } }
+    let(:ndbn_member_id) { 200040 }
+    let(:account_name) { "New Super Baby" }
+    let(:fake_html_body) do
+      <<-HTML
+      <table class="contentpaneopen">
+      <tbody><tr>
+      <td valign="top"><div class="content-wrapper" style="width: 690px;">
+      <h1>Organizational Member IDs</h1>
+      <p><strong>ID&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;Account Name</strong></p>
+      <p><span style="font-size: 15px;">20040&nbsp;&nbsp;&nbsp; (914) Cares</span></p>
+      <p>#{ndbn_member_id}&nbsp;&nbsp;&nbsp; #{account_name}</p>
+
+      </div></td>
+      </tr>
+
+      </tbody></table>
+      HTML
+    end
+
+    before do
+      fake_http_response = double("fake_http_response", body: fake_html_body)
+      allow(described_class).to receive(:get).with(SyncNDBNMembers::NDBN_MEMBERS_PAGE).and_return(fake_http_response)
```

Comment:
Can we use WebMock for this?

Reply:
Is that preferred over this approach? What benefits do we get with Webmock?

Reply:
WebMock is a much fuller-featured solution. In this case we probably don't gain all that much, but if we already have a framework designed to mock out HTTP requests, I feel like we probably should make use of it :)

Reply:
That makes sense, and also thinking about it I think we can benefit from testing out another failure mode of a non 200 response code.